Bottom: e2ed6a6d005ac9abc47299f9e063a92881757be7
Top:    fb2205e8060ae8498af1c3a7f9a85792e3a59766
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 15:20:52 +0100

Remove commented TEXT_IS_VISIBLE implementation

I'm cleaning up code which uses SWAP_IDENT, and this comment scored a hit.


---

diff --git a/src/macro.h b/src/macro.h
index 7cba5ac..19f80b8 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -151,12 +151,7 @@ extern int mem_any_set (unsigned char *, int);
 
 /* ---------------------------------------------------------------------------
  *  Determines if text is actually visible
-#define TEXT_IS_VISIBLE(b, l, t)					\
-		(((l)->On && !TEST_FLAG(ONSILKFLAG, (t))) ||     	\
-		(TEST_FLAG(ONSILKFLAG, (t)) && (b)->ElementOn   	\
-		&& ((b)->InvisibleObjectsOn ||                  	\
-		(TEST_FLAG(ONSOLDERFLAG, (t)) != 0) == SWAP_IDENT)))
-*/
+ */
 #define TEXT_IS_VISIBLE(b, l, t) \
 	((l)->On)
